src/widgets/widget_definitions.rs

```diff
@@ -230,7 +233,7 @@ fn build_gtk_color_chooser(bargs: &mut BuilderArgs) -> Result<gtk::ColorChooserW
 }
 
 /// @widget scale extends range
-/// @desc a slider.
+/// @desc A widget with a slider.
```

Comment:
It is just simply the slider tho :o
```suggestion
/// @desc A slider.
```

Reply:
🥇